web/src/components/tasks/business-formation/business/IsVeteranNonprofit.tsx

```diff
@@ -35,12 +35,13 @@ export const IsVeteranNonprofit = (): ReactElement => {
           row
         >
           <FormControlLabel
+            style={{ marginRight: "0px" }}
```

Comment:
We try to avoid `style` tags unless there's a very good reason to, and we use the USWDS classname style instead. We should use `margin-right-0` or similar instead of a style override.